src/main/java/frc/robot/subsystems/shooter/ShooterIOSim.java

```diff
@@ -0,0 +1,86 @@
+package frc.robot.subsystems.shooter;
+
+import com.ctre.phoenix6.controls.VelocityVoltage;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.controller.SimpleMotorFeedforward;
+import edu.wpi.first.units.Angle;
+import edu.wpi.first.units.MutableMeasure;
+import edu.wpi.first.units.Units;
+import edu.wpi.first.units.Velocity;
+import edu.wpi.first.wpilibj.Timer;
+import lib.motors.TalonFXSim;
+
+public class ShooterIOSim implements ShooterIO {
+    private final TalonFXSim topMotor;
+    private final TalonFXSim bottomMotor;
```

Comment:
Will be good to explicitly config neutral mode to "coast".

Reply:
how do i config neutral mode in a simulation?

Reply:
So... now I remember you don't use Phoenix 6's `TalonFXSimState` but your own class. If you would have used Phoenix 6's class, you'd have a regular `TalonFX` object which you could configure the same way you'd configure a "real" Talon FX. Maybe this is something we can work on next off-season. For now, resolving.